### ISSUE TITLE
Discriminator network not training

### ISSUE THREAD

**Issue by generic-github-user** [generic-github-user/xkcd-Generator#18]
The discriminator neural network does not train when generator network training is disabled. This should not be happening.

**Comment by generic-github-user** [comment]
Discriminator network score stuck near 1.

**Comment by generic-github-user** [comment]
Solved by increasing the learning rate and changing the activation function from "relu" to "tanh".

**Comment by generic-github-user** [comment]
Tanh seems to favor the discriminator, while sigmoid favors the generator.
